src/safeds/ml/classical/regression/_arima.py

```diff
@@ -70,9 +71,10 @@ def fit(self, time_series: TimeSeries) -> ArimaModelRegressor:
         """
         from statsmodels.tsa.arima.model import ARIMA
 
-        if not isinstance(time_series, TimeSeries) and isinstance(time_series, Table):
-            raise NonTimeSeriesError
-        if time_series.number_of_rows == 0:
+        if not isinstance(time_series, TimeSeriesDataset):
+            raise NonTimeSeriesDatasetError
```

Comment:
Is this needed as the `time_series` value has to be of type `TimeSeriesDataset` as described in the signature?

Reply:
In theory the User could pass a other Dataset, because the signature does not forece it. But the ARIMA model is only useful on timeseries data

Reply:
removed, as signature is enough